src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -448,7 +457,7 @@ struct areatrigger_pri_divine_star : AreaTriggerAI
                 returnSplinePoints.push_back(PositionToVector3(caster));
                 returnSplinePoints.push_back(PositionToVector3(caster));
 
-                at->InitSplines(returnSplinePoints, at->GetDistance(caster) / 24 * 1000);
+                at->InitSplines(returnSplinePoints, uint32(at->GetDistance(caster) / 24 * 1000));
```

Comment:
is the 24 in this line correct ?

Reply:
oops I need to obtain the data again, thanks for pointing out.